### PR TITLE
DCOS_OSS-5406: adding windows-instance module, AzureRM provider

### PR DESCRIPTION
**Pull request by sergiimatus**
@sebbrandt87 , done as per requested at https://github.com/dcos-terraform/terraform-azurerm-windows-instance/pull/1

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  virtual_machine_name = "${format(var.hostname_format, count.index + 1, local.cluster_name)}"
+  publisher            = "Microsoft.Compute"
+  type                 = "CustomScriptExtension"
+  type_handler_version = "1.8"
+  count                = "${var.num}"
+  depends_on           = ["azurerm_virtual_machine.windows_instance"]
+
+  settings = <<SETTINGS
+    {
+        "fileUris": ["https://raw.githubusercontent.com/dcos-terraform/terraform-azurerm-windows-instance/master/winrm_setup.ps1"],
```
Comment (sebbrandt87) [inline main.tf]: also as said in #1 
please use the file operation here for commandToExecute as we could not bet on github being available and want exactly the version of the script we package in each specific terraform module release.

**Review by sebbrandt87 — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (sergiimatus) [reply main.tf]: Tried `"fileUris": ["file(${path.module}/winrm_script.ps1)"]` and it doesn't work . 
Reason: as per [Microsoft documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-windows) - "Scripts can be downloaded from Azure storage or GitHub, or provided to the Azure portal at extension run time." Meaning, solution here is either to place content of winrm script using `cmd` syntax and execute line by line, or use as is - via github link, as shown in the example

Comment (sebbrandt87) [reply main.tf]: I see but this is a no go, as it is not bound to a released terraform module then but to a github ref that may change the whole content.
option would to add it via cmd and read by `template` or `file` to be interpreted via terraform.

**Review by sebbrandt87 — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (sergiimatus) [reply main.tf]: removed `fileUris` permanently
added `commandToExecute` with all command lines at `winrm_setup.ps1`
removed `winrm_setup.ps1`

**Review by sebbrandt87 — APPROVED** [review]
/lgtm

**Pull request merged**